packages/cli/src/commands/new/entity.ts

```diff
@@ -60,10 +61,13 @@ const run = async (name: string, rawFields: Array<string>, rawEvents: Array<stri
     .done()
 
 function generateImports(info: EntityInfo): Array<ImportDeclaration> {
-  const eventsImports: Array<ImportDeclaration> = info.events.map((eventData) => ({
-    packagePath: `../events/${eventData.eventName}`,
-    commaSeparatedComponents: eventData.eventName,
-  }))
+  const eventsImports: Array<ImportDeclaration> = info.events.map((eventData) => {
+    const fileName = inflection.underscore(eventData.eventName).replace(/_/g, '-')
```

Comment:
Oh! When I saw this line I thought that `inflection.dasherize` should do the job better, but I tried and... surprise! It doesn't! It looks like other people had the same issue and there are even PRs to solve it, but the repository seems abandoned https://github.com/dreamerslab/node.inflection/pull/72

You could avoid manually replacing underscores with dashes doing something like this:

```suggestion
    const fileName = inflection.transform( eventData.eventName, [ 'underscore', 'dasherize' ])
```

But maybe a better choice would be to try the library `inflected` (https://www.npmjs.com/package/inflected), which looks like a more up to date drop-in replacement. If you want to check if it works better, it could make sense to replace it in the whole project... (you'll likely need to do some other changes in other files, I bet the compiler will guide you, but if you need help, don't hesitate to ask in Discord 😉)

Reply:
Hi @javiertoledo ! I can't find the "transform" function so I use "dasherize" and "underscore". You can see it here: https://github.com/boostercloud/booster/blob/5069a978f11be9c41c0602e261b634a7c0e25a2f/packages/cli/src/commands/new/entity.ts#L65